dockers/docker-orchagent/supervisor-proc-exit-listener

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import os
+import signal
+import sys
+import syslog
+
+from supervisor import childutils
+
+# Process names as defined in supervisor.conf file
+critical_processes = [
+    'orchagent',
+    'portsyncd',
+    'intfsyncd',
+    'neighsyncd',
+    'vlanmgrd',
+    'intfmgrd',
+    'buffermgrd'
+]
```

Comment:
we should put this script as a SCRIPT so it can be shared by all dockers, then the process list should be move to a separate file instead of being hard-coded in the script.

Reply:
@lguohan: Where do you suggest we store the event listener script in the repo? We currently don't have files which are shared by multiple containers.

We could potentially add the script to the docker-base image and have it source a file containing the list of critical processes, but this might not be very straightforward.

Reply:
check rules/script.mk the CONFIGDB_LOAD_SCRIPT, it is shared by multiple containers.

Reply:
Done in commit https://github.com/Azure/sonic-buildimage/pull/2546/commits/b4c0ab2b26b5e184e62c03b8b1756a2ea54922cb